zhaquirks/smartwings/wm25lz.py

```diff
@@ -0,0 +1,105 @@
+"""Device handler for Smartwings blinds."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class InvertedWindowCoveringCluster(CustomCluster, WindowCovering):
+    """WindowCovering cluster implementation.
+
+    This implementation inverts the reported covering percent for non standard
+    devices that don't follow the reporting spec.
+    """
+
+    CMD_GO_UP = WindowCovering.commands_by_name["down_close"].id
+    CMD_GO_DOWN = WindowCovering.commands_by_name["up_open"].id
```

Comment:
Looking from the ZCL side, I'd have probably named the variables `CMD_GO_UP` = `up_open` (and vice versa).
But I guess yours is fine too when looking from the "blind side".